src/mantidprofiler/diskrecord.py

```diff
@@ -72,6 +77,41 @@ def monitor(pid: int, logfile: Path, interval: Optional[float], show_bytes: bool
                         conversion_to_size * (disk_after.write_bytes - disk_before.write_bytes) / delta_time
                     )
 
+                    # get information from children
+                    update_children(children, all_children(process))
+                    for key, child in children.items():
```

Comment:
`key` is not used in this loop. This can be replaced with 
```suggestion
                    for child in children.values():
```